src/ui/public/typeahead/_input.js

```diff
@@ -20,29 +20,22 @@ typeahead.directive('kbnTypeaheadInput', function ($rootScope) {
 
       // handle keypresses
       $el.on('keydown', function (ev) {
-        typeaheadCtrl.keypressHandler(ev);
-        digest();
+        $scope.$apply(() => typeaheadCtrl.keypressHandler(ev));
```

Comment:
I think `$evalAsync` might be better than `$apply` since it wouldn't trigger additional digest cycles if one is already in progress.

Reply:
f39629000